python/mkdocs/docs/api.md

```diff
@@ -200,7 +105,7 @@ table = StaticTable.from_metadata(
 
 ## Create a table
 
-To create a table from a catalog:
+To create a table by passing the table to the catalog:
```

Comment:
I'm not sure what you mean here by "passing the table to the catalog"?

Reply:
In between changes :) thanks!